tests/test_mariadb.py

```diff
@@ -0,0 +1,141 @@
+"""Tests for the MariaDB related application container images."""
+from itertools import product
+from typing import List
+from typing import Optional
+
+import pymysql
+import pytest
+from _pytest.mark import ParameterSet
+from pytest_container.container import container_from_pytest_param
+from pytest_container.container import ContainerData
+from pytest_container.container import DerivedContainer
+from tenacity import retry
+from tenacity import stop_after_attempt
+from tenacity import wait_exponential
+
+from bci_tester.data import MARIADB_CONTAINERS
+from bci_tester.data import MARIADB_ROOT_PASSWORD
+
+
+CONTAINER_IMAGES = MARIADB_CONTAINERS
+
+
+def test_entry_point(auto_container: ContainerData) -> None:
+    # do really nothing here, just check that the container launched
+    assert auto_container.connection.run_expect([0], "ps")
```

Comment:
You don't really need this, if the container does not launch, then the test setup will fail.